### PR TITLE
fixes console errors

### LINKED ISSUE #29: JS console errors

### PR DESCRIPTION
**Pull request by carl-alberto**
Fixes #29

### CODE REVIEW

File: assets/js/admin.min.js [hunk 1]
```diff
@@ -1,33 +1 @@
...
-		var page3 = url3.searchParams.get("page");
-		var tab3 = url3.searchParams.get("tab");
-		var updated3 = url3.searchParams.get("settings-updated");
-		if ( ( 'easy_symlinks_settings' === page3 ) && ( 'delete' === tab ) && ( 'true' === updated3 ) ) {
-			var redir = url.origin + '/wp-admin/options-general.php?page=easy_symlinks_settings&tab=delete';
-			console.log( redir );
-			window.location.replace( redir );
-		}
-	}
-);
```
Comment (wpugphil) [inline assets/js/admin.min.js]: :warning: **Warning**: File appears to be minified and cannot be processed (*Internal.Tokenizer.Exception*).

File: assets/js/admin.js [hunk 2]
```diff
@@ -26,7 +34,6 @@ jQuery(document).ready(
 		var updated3 = url3.searchParams.get("settings-updated");
```
Comment (wpugphil) [inline assets/js/admin.js]: :no_entry_sign: **Error**: Expected 1 spaces before closing bracket; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceBeforeCloseBracket*).

Comment (wpugphil) [inline assets/js/admin.js]: :no_entry_sign: **Error**: Expected 1 spaces after opening bracket; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceAfterOpenBracket*).

File: assets/js/admin.js [hunk 3]
```diff
@@ -15,8 +15,16 @@ jQuery(document).ready(
...
-				document.getElementById('link').value = '';
+
+				var caesTarget = document.getElementById('target');
+				if ( caesTarget ) {
+					document.getElementById('target').value = '';
+				}
+
+				var caesLink = document.getElementById('link');
+				if ( caesLink ) {
+					document.getElementById('link').value = '';
```
Comment (wpugphil) [inline assets/js/admin.js]: :no_entry_sign: **Error**: Expected 1 spaces before closing bracket; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceBeforeCloseBracket*).

Comment (wpugphil) [inline assets/js/admin.js]: :no_entry_sign: **Error**: Expected 1 spaces after opening bracket; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceAfterOpenBracket*).

File: assets/js/admin.js [hunk 4]
```diff
@@ -15,8 +15,16 @@ jQuery(document).ready(
...
 			if ( ( null === tab ) || ( 'add' === tab ) ) {
-				document.getElementById('target').value = '';
-				document.getElementById('link').value = '';
+
+				var caesTarget = document.getElementById('target');
+				if ( caesTarget ) {
+					document.getElementById('target').value = '';
+				}
+
+				var caesLink = document.getElementById('link');
```
Comment (wpugphil) [inline assets/js/admin.js]: :no_entry_sign: **Error**: Expected 1 spaces before closing bracket; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceBeforeCloseBracket*).

Comment (wpugphil) [inline assets/js/admin.js]: :no_entry_sign: **Error**: Expected 1 spaces after opening bracket; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceAfterOpenBracket*).

**Review by wpugphil — CHANGES_REQUESTED** [review]
**phpcs** scanning turned up:

:no_entry_sign: 10 errors 

:warning: 1 warning 

***

**hashes-api**-scanning skipped


Posting will continue in further review(s)

File: assets/js/admin.js [hunk 5]
```diff
@@ -15,8 +15,16 @@ jQuery(document).ready(
 
 		if ( 'easy_symlinks_settings' === page ) {
 			if ( ( null === tab ) || ( 'add' === tab ) ) {
-				document.getElementById('target').value = '';
-				document.getElementById('link').value = '';
+
+				var caesTarget = document.getElementById('target');
+				if ( caesTarget ) {
+					document.getElementById('target').value = '';
```
Comment (wpugphil) [inline assets/js/admin.js]: :no_entry_sign: **Error**: Expected 1 spaces before closing bracket; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceBeforeCloseBracket*).

Comment (wpugphil) [inline assets/js/admin.js]: :no_entry_sign: **Error**: Expected 1 spaces after opening bracket; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceAfterOpenBracket*).

File: assets/js/admin.js [hunk 6]
```diff
@@ -15,8 +15,16 @@ jQuery(document).ready(
 
 		if ( 'easy_symlinks_settings' === page ) {
 			if ( ( null === tab ) || ( 'add' === tab ) ) {
-				document.getElementById('target').value = '';
-				document.getElementById('link').value = '';
+
+				var caesTarget = document.getElementById('target');
```
Comment (wpugphil) [inline assets/js/admin.js]: :no_entry_sign: **Error**: Expected 1 spaces before closing bracket; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceBeforeCloseBracket*).

**Review by wpugphil — CHANGES_REQUESTED** [review]
Previous scan continued.

File: assets/js/admin.js (hunk 6 above)
Comment (wpugphil) [inline assets/js/admin.js]: :no_entry_sign: **Error**: Expected 1 spaces after opening bracket; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceAfterOpenBracket*).

File: assets/js/admin.min.js [hunk 7]
```diff
@@ -8,25 +8,32 @@ jQuery(document).ready(
...
+				}
 			}
 		}
 
 		var url3 = new URL(url.origin + refer);
-		var page3 = url3.searchParams.get("page");
-		var tab3 = url3.searchParams.get("tab");
-		var updated3 = url3.searchParams.get("settings-updated");
+		var page3 = url3.searchParams.get( 'page' );
+		var tab3 = url3.searchParams.get( 'tab' );
```
Comment (wpugphil) [inline assets/js/admin.min.js]: :warning: **Warning**: Equals sign not aligned with surrounding assignments; expected 5 spaces but found 1 space (*Generic.Formatting.MultipleStatementAlignment.NotSameWarning*).

File: assets/js/admin.min.js [hunk 8]
```diff
@@ -8,25 +8,32 @@ jQuery(document).ready(
...
+					document.getElementById( 'link' ).value = '';
+				}
 			}
 		}
 
 		var url3 = new URL(url.origin + refer);
-		var page3 = url3.searchParams.get("page");
-		var tab3 = url3.searchParams.get("tab");
-		var updated3 = url3.searchParams.get("settings-updated");
+		var page3 = url3.searchParams.get( 'page' );
```
Comment (wpugphil) [inline assets/js/admin.min.js]: :warning: **Warning**: Equals sign not aligned with surrounding assignments; expected 4 spaces but found 1 space (*Generic.Formatting.MultipleStatementAlignment.NotSameWarning*).

File: assets/js/admin.min.js [hunk 9]
```diff
@@ -8,25 +8,32 @@ jQuery(document).ready(
...
+				}
+
+				var caesLink = document.getElementById( 'link' );
+				if ( caesLink ) {
+					document.getElementById( 'link' ).value = '';
+				}
 			}
 		}
 
 		var url3 = new URL(url.origin + refer);
```
Comment (wpugphil) [inline assets/js/admin.min.js]: :no_entry_sign: **Error**: Expected 1 spaces before closing bracket; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceBeforeCloseBracket*).

Comment (wpugphil) [inline assets/js/admin.min.js]: :no_entry_sign: **Error**: Expected 1 spaces after opening bracket; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceAfterOpenBracket*).

Comment (wpugphil) [inline assets/js/admin.min.js]: :warning: **Warning**: Equals sign not aligned with surrounding assignments; expected 5 spaces but found 1 space (*Generic.Formatting.MultipleStatementAlignment.NotSameWarning*).

File: assets/js/admin.min.js [hunk 10]
```diff
@@ -8,25 +8,32 @@ jQuery(document).ready(
...
 		var url_string = window.location.href;
 		var url = new URL(url_string);
-		var page = url.searchParams.get("page");
-		var tab = url.searchParams.get("tab");
-		var deleted = url.searchParams.get('caes_deleted');
-		var refer = document.getElementsByName("_wp_http_referer")[0].value;
+		var page = url.searchParams.get( 'page' );
+		var tab = url.searchParams.get( 'tab');
+		var deleted = url.searchParams.get( 'caes_deleted') ;
+		var refer = document.getElementsByName( '_wp_http_referer' )[0].value;
```
Comment (wpugphil) [inline assets/js/admin.min.js]: :warning: **Warning**: Equals sign not aligned with surrounding assignments; expected 6 spaces but found 1 space (*Generic.Formatting.MultipleStatementAlignment.NotSameWarning*).

File: assets/js/admin.min.js [hunk 11]
```diff
@@ -8,25 +8,32 @@ jQuery(document).ready(
 	function (e) {
 		var url_string = window.location.href;
 		var url = new URL(url_string);
-		var page = url.searchParams.get("page");
-		var tab = url.searchParams.get("tab");
-		var deleted = url.searchParams.get('caes_deleted');
-		var refer = document.getElementsByName("_wp_http_referer")[0].value;
+		var page = url.searchParams.get( 'page' );
+		var tab = url.searchParams.get( 'tab');
+		var deleted = url.searchParams.get( 'caes_deleted') ;
```
Comment (wpugphil) [inline assets/js/admin.min.js]: :no_entry_sign: **Error**: Space found before semicolon; expected &quot;);&quot; but found &quot;) ;&quot; (*Squiz.WhiteSpace.SemicolonSpacing.Incorrect*).

**Review by wpugphil — CHANGES_REQUESTED** [review]
**phpcs** scanning turned up:

:no_entry_sign: 16 errors 

:warning: 16 warnings 

***

**hashes-api**-scanning skipped


Posting will continue in further review(s)

File: assets/js/admin.min.js (hunk 11 above)
Comment (wpugphil) [inline assets/js/admin.min.js]: :no_entry_sign: **Error**: Space after closing parenthesis of function call prohibited (*PEAR.Functions.FunctionCallSignature.SpaceAfterCloseBracket*).

Comment (wpugphil) [inline assets/js/admin.min.js]: :no_entry_sign: **Error**: Expected 1 spaces before closing bracket; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceBeforeCloseBracket*).

Comment (wpugphil) [inline assets/js/admin.min.js]: :warning: **Warning**: Equals sign not aligned with surrounding assignments; expected 4 spaces but found 1 space (*Generic.Formatting.MultipleStatementAlignment.NotSameWarning*).

File: assets/js/admin.min.js [hunk 12]
```diff
@@ -8,25 +8,32 @@ jQuery(document).ready(
 	function (e) {
 		var url_string = window.location.href;
 		var url = new URL(url_string);
-		var page = url.searchParams.get("page");
-		var tab = url.searchParams.get("tab");
-		var deleted = url.searchParams.get('caes_deleted');
-		var refer = document.getElementsByName("_wp_http_referer")[0].value;
+		var page = url.searchParams.get( 'page' );
+		var tab = url.searchParams.get( 'tab');
```
Comment (wpugphil) [inline assets/js/admin.min.js]: :no_entry_sign: **Error**: Expected 1 spaces before closing bracket; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceBeforeCloseBracket*).

Comment (wpugphil) [inline assets/js/admin.min.js]: :warning: **Warning**: Equals sign not aligned with surrounding assignments; expected 8 spaces but found 1 space (*Generic.Formatting.MultipleStatementAlignment.NotSameWarning*).

File: assets/js/admin.min.js [hunk 13]
```diff
@@ -8,25 +8,32 @@ jQuery(document).ready(
 	function (e) {
 		var url_string = window.location.href;
 		var url = new URL(url_string);
-		var page = url.searchParams.get("page");
-		var tab = url.searchParams.get("tab");
-		var deleted = url.searchParams.get('caes_deleted');
-		var refer = document.getElementsByName("_wp_http_referer")[0].value;
+		var page = url.searchParams.get( 'page' );
```
Comment (wpugphil) [inline assets/js/admin.min.js]: :warning: **Warning**: Equals sign not aligned with surrounding assignments; expected 7 spaces but found 1 space (*Generic.Formatting.MultipleStatementAlignment.NotSameWarning*).

File: assets/js/admin.min.js [hunk 14]
```diff
@@ -8,25 +8,32 @@ jQuery(document).ready(
 	function (e) {
 		var url_string = window.location.href;
 		var url = new URL(url_string);
```
Comment (wpugphil) [inline assets/js/admin.min.js]: :no_entry_sign: **Error**: Expected 1 spaces before closing bracket; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceBeforeCloseBracket*).

Comment (wpugphil) [inline assets/js/admin.min.js]: :no_entry_sign: **Error**: Expected 1 spaces after opening bracket; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceAfterOpenBracket*).

Comment (wpugphil) [inline assets/js/admin.min.js]: :warning: **Warning**: Equals sign not aligned with surrounding assignments; expected 8 spaces but found 1 space (*Generic.Formatting.MultipleStatementAlignment.NotSameWarning*).

File: assets/js/admin.js [hunk 15]
```diff
@@ -8,25 +8,32 @@ jQuery(document).ready(
...
+				}
 			}
 		}
 
 		var url3 = new URL(url.origin + refer);
-		var page3 = url3.searchParams.get("page");
-		var tab3 = url3.searchParams.get("tab");
-		var updated3 = url3.searchParams.get("settings-updated");
+		var page3 = url3.searchParams.get( 'page' );
+		var tab3 = url3.searchParams.get( 'tab' );
```
Comment (wpugphil) [inline assets/js/admin.js]: :warning: **Warning**: Equals sign not aligned with surrounding assignments; expected 5 spaces but found 1 space (*Generic.Formatting.MultipleStatementAlignment.NotSameWarning*).

File: assets/js/admin.js [hunk 16]
```diff
@@ -8,25 +8,32 @@ jQuery(document).ready(
...
+					document.getElementById( 'link' ).value = '';
+				}
 			}
 		}
 
 		var url3 = new URL(url.origin + refer);
-		var page3 = url3.searchParams.get("page");
-		var tab3 = url3.searchParams.get("tab");
-		var updated3 = url3.searchParams.get("settings-updated");
+		var page3 = url3.searchParams.get( 'page' );
```
Comment (wpugphil) [inline assets/js/admin.js]: :warning: **Warning**: Equals sign not aligned with surrounding assignments; expected 4 spaces but found 1 space (*Generic.Formatting.MultipleStatementAlignment.NotSameWarning*).

**Review by wpugphil — CHANGES_REQUESTED** [review]
Previous scan continued.

File: assets/js/admin.js [hunk 17]
```diff
@@ -8,25 +8,32 @@ jQuery(document).ready(
...
+				}
+
+				var caesLink = document.getElementById( 'link' );
+				if ( caesLink ) {
+					document.getElementById( 'link' ).value = '';
+				}
 			}
 		}
 
 		var url3 = new URL(url.origin + refer);
```
Comment (wpugphil) [inline assets/js/admin.js]: :no_entry_sign: **Error**: Expected 1 spaces before closing bracket; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceBeforeCloseBracket*).

Comment (wpugphil) [inline assets/js/admin.js]: :no_entry_sign: **Error**: Expected 1 spaces after opening bracket; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceAfterOpenBracket*).

Comment (wpugphil) [inline assets/js/admin.js]: :warning: **Warning**: Equals sign not aligned with surrounding assignments; expected 5 spaces but found 1 space (*Generic.Formatting.MultipleStatementAlignment.NotSameWarning*).

File: assets/js/admin.js [hunk 18]
```diff
@@ -8,25 +8,32 @@ jQuery(document).ready(
...
 		var url_string = window.location.href;
 		var url = new URL(url_string);
-		var page = url.searchParams.get("page");
-		var tab = url.searchParams.get("tab");
-		var deleted = url.searchParams.get('caes_deleted');
-		var refer = document.getElementsByName("_wp_http_referer")[0].value;
+		var page = url.searchParams.get( 'page' );
+		var tab = url.searchParams.get( 'tab');
+		var deleted = url.searchParams.get( 'caes_deleted') ;
+		var refer = document.getElementsByName( '_wp_http_referer' )[0].value;
```
Comment (wpugphil) [inline assets/js/admin.js]: :warning: **Warning**: Equals sign not aligned with surrounding assignments; expected 6 spaces but found 1 space (*Generic.Formatting.MultipleStatementAlignment.NotSameWarning*).

File: assets/js/admin.js [hunk 19]
```diff
@@ -8,25 +8,32 @@ jQuery(document).ready(
 	function (e) {
 		var url_string = window.location.href;
 		var url = new URL(url_string);
-		var page = url.searchParams.get("page");
-		var tab = url.searchParams.get("tab");
-		var deleted = url.searchParams.get('caes_deleted');
-		var refer = document.getElementsByName("_wp_http_referer")[0].value;
+		var page = url.searchParams.get( 'page' );
+		var tab = url.searchParams.get( 'tab');
+		var deleted = url.searchParams.get( 'caes_deleted') ;
```
Comment (wpugphil) [inline assets/js/admin.js]: :no_entry_sign: **Error**: Space found before semicolon; expected &quot;);&quot; but found &quot;) ;&quot; (*Squiz.WhiteSpace.SemicolonSpacing.Incorrect*).

Comment (wpugphil) [inline assets/js/admin.js]: :no_entry_sign: **Error**: Space after closing parenthesis of function call prohibited (*PEAR.Functions.FunctionCallSignature.SpaceAfterCloseBracket*).

**Review by wpugphil — CHANGES_REQUESTED** [review]
Previous scan continued.

File: assets/js/admin.js (hunk 19 above)
Comment (wpugphil) [inline assets/js/admin.js]: :no_entry_sign: **Error**: Expected 1 spaces before closing bracket; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceBeforeCloseBracket*).

Comment (wpugphil) [inline assets/js/admin.js]: :warning: **Warning**: Equals sign not aligned with surrounding assignments; expected 4 spaces but found 1 space (*Generic.Formatting.MultipleStatementAlignment.NotSameWarning*).

File: assets/js/admin.js [hunk 20]
```diff
@@ -8,25 +8,32 @@ jQuery(document).ready(
 	function (e) {
 		var url_string = window.location.href;
 		var url = new URL(url_string);
-		var page = url.searchParams.get("page");
-		var tab = url.searchParams.get("tab");
-		var deleted = url.searchParams.get('caes_deleted');
-		var refer = document.getElementsByName("_wp_http_referer")[0].value;
+		var page = url.searchParams.get( 'page' );
+		var tab = url.searchParams.get( 'tab');
```
Comment (wpugphil) [inline assets/js/admin.js]: :no_entry_sign: **Error**: Expected 1 spaces before closing bracket; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceBeforeCloseBracket*).

Comment (wpugphil) [inline assets/js/admin.js]: :warning: **Warning**: Equals sign not aligned with surrounding assignments; expected 8 spaces but found 1 space (*Generic.Formatting.MultipleStatementAlignment.NotSameWarning*).

File: assets/js/admin.js [hunk 21]
```diff
@@ -8,25 +8,32 @@ jQuery(document).ready(
 	function (e) {
 		var url_string = window.location.href;
 		var url = new URL(url_string);
-		var page = url.searchParams.get("page");
-		var tab = url.searchParams.get("tab");
-		var deleted = url.searchParams.get('caes_deleted');
-		var refer = document.getElementsByName("_wp_http_referer")[0].value;
+		var page = url.searchParams.get( 'page' );
```
Comment (wpugphil) [inline assets/js/admin.js]: :warning: **Warning**: Equals sign not aligned with surrounding assignments; expected 7 spaces but found 1 space (*Generic.Formatting.MultipleStatementAlignment.NotSameWarning*).

File: assets/js/admin.js [hunk 22]
```diff
@@ -8,25 +8,32 @@ jQuery(document).ready(
 	function (e) {
 		var url_string = window.location.href;
 		var url = new URL(url_string);
```
Comment (wpugphil) [inline assets/js/admin.js]: :no_entry_sign: **Error**: Expected 1 spaces before closing bracket; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceBeforeCloseBracket*).

**Review by wpugphil — CHANGES_REQUESTED** [review]
Previous scan continued.

File: assets/js/admin.js (hunk 22 above)
Comment (wpugphil) [inline assets/js/admin.js]: :no_entry_sign: **Error**: Expected 1 spaces after opening bracket; 0 found (*PEAR.Functions.FunctionCallSignature.SpaceAfterOpenBracket*).

Comment (wpugphil) [inline assets/js/admin.js]: :warning: **Warning**: Equals sign not aligned with surrounding assignments; expected 8 spaces but found 1 space (*Generic.Formatting.MultipleStatementAlignment.NotSameWarning*).

**Review by carl-alberto — COMMENTED** [review]
Fixes all

**Pull request merged**